src/lib/operators/import.cpp

```diff
@@ -51,9 +53,37 @@ std::shared_ptr<const Table> Import::_on_execute() {
   auto table = std::shared_ptr<Table>{};
 
   switch (_file_type) {
-    case FileType::Csv:
-      table = CsvParser::parse(filename, _chunk_size, _csv_meta);
+    case FileType::Csv: {
+      if (static_cast<int>(static_cast<bool>(_csv_meta)) +
+              static_cast<int>(Hyrise::get().storage_manager.has_table(_tablename)) +
+              static_cast<int>(std::filesystem::exists(filename + CsvMeta::META_FILE_EXTENSION)) >=
+          2) {
+        Fail(
+            "Cannot load table from csv. Table definition source is ambiguous. At least two of the following sources "
+            "gave CsvMeta information: Import; existing table in DBMS; Meta file next to .csv file");
+      }
+
+      CsvMeta csv_meta;
+      if (_csv_meta) {
+        csv_meta = *_csv_meta;
+      } else if (Hyrise::get().storage_manager.has_table(_tablename)) {
+        TableColumnDefinitions column_definitions =
+            Hyrise::get().storage_manager.get_table(_tablename)->column_definitions();
+        csv_meta.columns.resize(column_definitions.size());
+        for (TableColumnDefinitions::size_type i = 0; i < column_definitions.size(); i++) {
```

Comment:
```suggestion
        for (auto column_id = ColumnID{0}; column_id < column_count; ++column_id) {
```

Reply:
The `ColumnID` type is the best fit to iterate over columns.
Auto-to-stick is preferred here, too.
We don't want to call `.size()` in loops since it is called each iteration, better save the size as a constant before.
Prefer the pre increment because the post increment returns an unnecessary rvalue to a copy of the old value of the index.

Probably the compiler would optimize these things anyway, but let's be sure.